meta/store.go

```diff
@@ -394,6 +397,30 @@ func (s *Store) enableRemoteRaft() error {
 	return s.changeState(rr)
 }
 
+func (s *Store) checkRaftState() error {
```

Comment:
Not sure I understand what this func is doing.  Is it checking to see if it should enable raft locally?  If it's already enabled, do nothing?